### PR TITLE
Trim on edge hover

### PR DESCRIPTION
**Pull request by grliszas14**
Resolves: https://github.com/audacity/audacity/issues/7335

### CODE REVIEW

File: src/app/CMakeLists.txt [hunk 1]
```diff
@@ -104,7 +104,9 @@ endif()
 ###########################################
 # Resources
 ###########################################
-qt_add_resources(APP_RCC_SOURCES app.qrc)
+qt_add_resources(APP_RCC_SOURCES
+    app.qrc
+    ${PROJECT_SOURCE_DIR}/images/resources.qrc)
```
Comment (Eism) [inline src/app/CMakeLists.txt:109]: hmm, can we move it to `projectscene` module?

Comment (grliszas14) [reply src/app/CMakeLists.txt]: fixed 👍

File: src/projectscene/view/clipsview/clipslistmodel.cpp [hunk 2]
```diff
@@ -301,6 +301,30 @@ bool ClipsListModel::changeClipTitle(const ClipKey& key, const QString& newTitle
     return ok;
 }
 
+QVariant ClipsListModel::getSortedNeighbor(const ClipKey& key, int offset) const
+{
+    QList<ClipListItem*> sortedList = m_clipList;
+
+    std::sort(sortedList.begin(), sortedList.end(), [](ClipListItem* a, ClipListItem* b) {
```
Comment (igorkorsukov) [inline src/projectscene/view/clipsview/clipslistmodel.cpp:308]: I think we can do sort once in the `reload` method.

Comment (grliszas14) [reply src/projectscene/view/clipsview/clipslistmodel.cpp]: moved sorting to update() function - that's the place where m_clipList (clips that are visible on a screen) get updated

File: src/projectscene/view/clipsview/clipslistmodel.cpp [hunk 3]
```diff
@@ -301,6 +305,26 @@ bool ClipsListModel::changeClipTitle(const ClipKey& key, const QString& newTitle
     return ok;
 }
 
+QVariant ClipsListModel::getSortedNeighbor(const ClipKey& key, int offset) const
+{
+    QList<ClipListItem*> sortedList = m_clipList;
```
Comment (igorkorsukov) [inline src/projectscene/view/clipsview/clipslistmodel.cpp:310]: now you don't need to make a copy

File: src/projectscene/view/clipsview/clipslistmodel.cpp [hunk 4]
```diff
@@ -301,6 +305,26 @@ bool ClipsListModel::changeClipTitle(const ClipKey& key, const QString& newTitle
     return ok;
 }
 
+QVariant ClipsListModel::getSortedNeighbor(const ClipKey& key, int offset) const
```
Comment (igorkorsukov) [inline src/projectscene/view/clipsview/clipslistmodel.cpp:308]: I think now we can just call it `neighbor`, or I think it's better to add two methods - the `next` and the `prev`

Comment (grliszas14) [reply src/projectscene/view/clipsview/clipslistmodel.cpp]: added next() and prev() methods + left neighbor() as a private member

**Review by Eism — APPROVED** [review]

**Pull request merged**